optuna/visualization/_edf.py

```diff
@@ -0,0 +1,141 @@
+import itertools
+from typing import List
+from typing import Sequence
+from typing import Union
+
+import numpy as np
+
+from optuna.logging import get_logger
+from optuna.study import Study
+from optuna.trial import TrialState
+from optuna.visualization._plotly_imports import _imports
+
+if _imports.is_successful():
+    from optuna.visualization._plotly_imports import go
+
+_logger = get_logger(__name__)
+
+
+def plot_edf(study: Union[Study, Sequence[Study]]) -> "go.Figure":
+    """Plot the objective value EDF (empirical distribution function) of a study.
+
+    Note that only the complete trials are considered when plotting the EDF.
+
+    .. note::
+
+        EDF is useful to analyze and improve search spaces.
+        For instance, you can see a practical use case of EDF in the paper
+        `Designing Network Design Spaces <ttps://arxiv.org/abs/2003.13678>`_.
+
```

Comment:
@sile Thanks for the reply. I think we can note the underlying distribution for the objective in the documentation as follows.
```suggestion

    .. note::

        The plotted EDF assumes that the value of the objective function is in 
        accordance with the uniform distribution over the objective space.
        
```

Reply:
Thanks!